frontend/src/components/Selector/Selector.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { SimpleSelect } from '@instructure/ui-simple-select/lib/SimpleSelect';
+
+const Selector = (props) => {
+  const { labelText, options } = props;
+  const [selectorValue, setSelectorValue] = useState();
+
+  return (
+    <SimpleSelect
+      renderLabel={labelText}
+      assistiveText="Use arrow keys to navigate options."
+      value={selectorValue}
+      onChange={(event) => setSelectorValue(event.target.value)}
```

Comment:
:) You can definitely file this in "premature optimization", but when the default event is all you need, the React docs (if I'm understanding them correctly) recommend defining a function e.g. in the component definition instead of using an inline func. (see https://reactjs.org/docs/handling-events.html, and search for "recommend")

Reply:
Good point. I should avoid writing inline functions.